hack/util.sh

```diff
@@ -458,10 +458,15 @@ function install_registry() {
 }
 readonly -f install_registry
 
-function wait_for_registry() {
+function is_registry_ready() {
 	local generation="$(oc get dc/docker-registry -o 'jsonpath={.metadata.generation}')"
 	local onereplicajs='{.status.observedGeneration},{.status.replicas},{.status.updatedReplicas},{.status.availableReplicas}'
-	wait_for_command "oc get dc/docker-registry -o 'jsonpath=${onereplicajs}' --config='${ADMIN_KUBECONFIG}' | grep '^${generation},1,1,1$'"  "$((5*TIME_MIN))"
+	oc get dc/docker-registry -o "jsonpath=${onereplicajs}" --config="${ADMIN_KUBECONFIG}" | fgrep -qx "${generation},1,1,1"
```

Comment:
> @stevekuznetsov I have an opposite question: why I shouldn't?

@php-coder 
 - change for the sake of change is not a good argument
 - `fgrep` is not preferred to `grep -F`

Justify your change by what it adds?